gist/client.py

```diff
@@ -175,11 +177,22 @@ def terminal_width():
 
     """
     try:
-        exitcode = fcntl.ioctl(
-                0,
-                termios.TIOCGWINSZ,
-                struct.pack('HHHH', 0, 0, 0, 0))
-        h, w, hp, wp = struct.unpack('HHHH', exitcode)
+        if platform.system() == "Windows":
+            from ctypes import windll, create_string_buffer
+            hStdErr = -12
```

Comment:
Why is this initialized to -12?

Reply:
This is the documented value on the [MSDN](https://docs.microsoft.com/en-us/windows/console/getstdhandle) for `stderr`

Reply:
Would you mind adding a comment to the code with that link?

Reply:
No problem